### DISCUSSION TITLE
Navigation problem

### DISCUSSION THREAD

**Discussion by huoranger** [jpanther/congo#205] [General]
I've built a blog through this theme, but navigation seems to be not working, I'm not sure what's going on? Ha ha
![image](https://user-images.githubusercontent.com/67517385/170855337-f0d95899-5bad-4a83-8a30-9f8d16223350.png)
![image](https://user-images.githubusercontent.com/67517385/170855346-980eca70-bec8-4376-a4af-a38d73e4e3e8.png)

**Comment by huoranger** [comment]
Clicking on any navigation takes you to the home page

**Reply by jpanther** [reply]
What version of Hugo are you using?

**Reply by iosonodenis** [reply]
Similar issue for me, maybe the same, maybe not.

When previewing locally with the command Hugo Server the menu are correctly working. When deploying with netlify the menus are not working.
More precisely, it's the href attribute not rendering correctly (it is just an empty href without any ="path")
My Hugo version is hugo v0.101.0+extended darwin/arm64 BuildDate=unknown

I have not try yet to publish locally with "Hugo", just to see if this is a netlify build issue

**Reply by jpanther** [reply]
The theme definitely works in this configuration with Netlify - it's how I use it myself. Are you setting the baseURL parameter? What build command are you using?

If you haven't already, be sure to check the [deployment documentation](https://jpanther.github.io/congo/docs/hosting-deployment/#netlify) for more details.

**Reply by iosonodenis** [reply]
I did set a baseurl and I set Hugo has a build command in netlify, but I did not set Congo as a module (manual install)
